Source/DafnyRuntime/DafnyRuntime.py

```diff
@@ -134,36 +132,25 @@ def __len__(self):
         return self.length(0)
 
     def __getitem__(self, key):
-        if isinstance(key, slice):
+        if isinstance(key, slice) or not isinstance(key, Iterable):
```

Comment:
```suggestion
        if not isinstance(key, Iterable):
```

I believe these two are equivalent